Source/Rhetos.Host.AspNet/RhetosAspNetCoreServiceCollectionExtensions.cs

```diff
@@ -18,23 +18,35 @@
 */
 
 using System;
+using Autofac;
 using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Rhetos;
 using Rhetos.Host.AspNet;
+using Rhetos.Logging;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     public static class RhetosAspNetCoreServiceCollectionExtensions
     {
-        public static RhetosAspNetServiceCollectionBuilder AddRhetos(this IServiceCollection serviceCollection, Action<IRhetosHostBuilder> configureRhetosHost = null)
+        public static RhetosAspNetServiceCollectionBuilder AddRhetos(
+            this IServiceCollection serviceCollection,
+            Action<IRhetosHostBuilder> configureRhetosHost = null,
+            Action<IServiceProvider, IRhetosHostBuilder> configureRhetosHostIntegration = null)

```

Comment:
Instead of having a second parameter here there are 2 other options:

1.  AddRhetos() with different signature
2. These configureActions are added via extension to RhetosAspNetServiceCollectionBuilder class

Not really sure if any of this is significant improvent (or at all), but having multiple optional parameters smells a bit.

Reply:
`Action<IRhetosHostBuilder>` is covered by `Action<IServiceProvider, IRhetosHostBuilder>`. Can we remove the first delegate?